plugin/src/test/kotlin/com/ritense/plugin/service/PluginServiceIT.kt

```diff
@@ -343,4 +345,37 @@ internal class PluginServiceIT : BaseIntegrationTest() {
             )
         }
     }
+
+    @Test
+    @Transactional
+    fun `should deploy plugin with resolved properties`() {
+        val pluginConfigurationId = UUID.randomUUID()
+        System.setProperty("MY_URL_PLACEHOLDER", "https://www.example.com/")
+        System.setProperty("MY_FIRST_PLACEHOLDER", "first_value")
+        System.setProperty("MY_SECOND_PLACEHOLDER", "second_value")
```

Comment:
Could you tests an `env` value as well?

Reply:
I've tried. Unfortunately, its very difficult to set an environment variable. Since it works differently for every system.